docs/overview/office-add-ins-community-call.md

```diff
@@ -17,20 +17,21 @@ The agenda for June will be posted about a week before the call.
-- [September 8, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-september-8-2021/ba-p/2747100)
-- [August 11, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-august-2021/ba-p/2661372)
-- [July 14, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-july-2021/ba-p/2573384)
-- [June 9, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-june-2021/ba-p/2446156)
-- [May 12, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-may-2021/ba-p/2369804)
-- [April 14, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-april-14-2021/ba-p/2318886)
-- [March 10, 2021](https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/office-add-ins-community-call-march-10-2021/ba-p/2205369)
+- [May 11, 2022](https://pnp.github.io/blog/office-add-ins-community-call/2022-05-11/)
+- [April 13, 2022](https://pnp.github.io/blog/office-add-ins-community-call/2022-04-13/)
+- [March 9, 2022](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-march-9-2022/)
+- [February 9, 2022](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-february-9-2022/)
+- [January 12, 2022](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-january-12-2022/)
+- [December 8, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-december-8-2021/)
+- [November 10, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-november-10-2021/)
+- [October 13, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-october-13-2021/)
+- [September 8, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-september-8-2021/)
+- [August 11, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-august-2021/)
+- [July 14, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-july-2021/)
+- [June 9, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-june-2021/)
+- [May 12, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-may-2021/)
+- [April 14, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-april-14-2021/)
+- [March 10, 2021](https://pnp.github.io/blog/office-add-ins-community-call/office-add-ins-community-call-march-10-2021/)
 - [Archived Office Add-ins community calls – June 10, 2020 - February 10, 2021](https://cdn.graph.office.net/prod/office/Office-Add-ins-Community-Call-Archive.pdf)
 
 ## See also
```

Comment:
Should we remove or update the link for 'Microsoft 365 PnP on Tech Community'?

Reply:
Yes, hang on will fix.